x-pack/plugins/timelines/public/components/t_grid/toolbar/fields_browser/field_table.tsx

```diff
@@ -25,6 +25,9 @@ export interface FieldTableProps {
    * the filter input (as a substring).
    */
   filteredBrowserFields: BrowserFields;
+  /** when true, show only the the selected field */
+  filterSelectedEnabled: boolean;
+  setFilterSelectedEnabled: (enabled: boolean) => void;
```

Comment:
@semd I’m not a huge fan of passing down the state methods over the props, do you think is better to have it as it is or maybe introduce the callback instead of `setFilterSelectedEnabled` ?

Reply:
Yes, I agree it is better to pass a callback instead of the state setter. done https://github.com/elastic/kibana/pull/128049/commits/810e7959914f3c152db3bbf5b4456411cc299f65